templates/nginx-confdnestedincludes-minio-publicscratch.j2

```diff
@@ -0,0 +1,4 @@
+location /publicscratch {
+    proxy_pass http://127.0.0.1:9000/publicscratch;
```

Comment:
I'm not overly convinced of the name "publicscratch" for use in demos, but no need to change it now.